apps/site/src/views/Landing/components/ApplyButton/ApplyButton.module.scss

```diff
@@ -0,0 +1,33 @@
+@use "bootstrap-utils" as utils;
+
+.applyButtonContainer {
+	position: relative;
+}
+
+.applyButton {
+	position: relative;
+	transform: skew(-30deg);
+	height: 80px;
+	width: 250px;
+	margin: 0px;
+	border: none;
+	background-color: utils.$yellow;
```

Comment:
How would you like me to make this change?

Reply:
Remove the `zothacks-theme` import at the top of `_bootstrap_utils.scss`. Then you can import the utils as `bootstrap` and then the ZotHacks theme as `theme`, so now it's more apparent that the colors are related to our theme instead of utils.